templates/taxi/driver_form.html

```diff
@@ -3,6 +3,7 @@
 
 {% block content %}
   <h1>Create driver</h1>
+  <h1>{{ object|yesno:"Update,Create" }} driver license</h1>
```

Comment:
The use of 'object|yesno:"Update,Create"' for the title may not work as intended if 'object' is not a boolean. Consider using a more explicit check to determine whether the form is for updating or creating a driver license.